docker/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Correcting the URL format
+URL=$(echo "$URL" | sed 's|/$||')  # Remove trailing slash if exists
+if ! echo "$URL" | grep -q "^https://"; then
+  URL="https://$URL"  # Add https:// if not present
+fi
```

Comment:
Is it possible a user might want to NOT run https?

Reply:
Ahh... you are correct, they might want http as well. What I found was if I didn't include this, streaming didn't work properly. Ideally I'd like to leave this part out and get streaming functioning regardless if the user enters http, https, or just the hostname or fqdn.

Going to remove this portion for now.

Reply:
Removed... what would be ideal, but I couldn't really figure out, is for flask to determine what the endpoint is requesting in terms of port and address. I know the URL or hostname that's entered is self referential and makes its way into how the gui routes it's interface.

NGinx can easily send this data along back to the host... but the re-write of how to make this work on the back end was beyond me.